pkg/tempopb/tempo.proto

```diff
@@ -58,11 +58,7 @@ message TraceByIDRequest {
 message TraceByIDResponse {
   Trace trace = 1;
   TraceByIDMetrics metrics = 2;
-  enum Status {
-    COMPLETE = 0;
-    PARTIAL = 1;
-  }
-  Status status = 3;
+  PartialStatus status = 3;
```

Comment:
nice work reusing this

Reply:
we don't have to do a V2 - it's just extra precautions for when we roll out and Grafana is not yet using the new protos.

Reply:
adding a field to a proto object does not break current readers. let's not add a `v2` of an endpoint unless we have to. can we update the PR to just change the current one?